hezronkimutai/questioner
# Ch set up folder structure 163087019

What does this PR do?

Creates the directory structure with the necessary files and packages for running the app.

Description of Tasks completed ??

Sets up the folder /package structure for versioning my apis and running packages
adds and init.py file to each module

How should this be manually tested?

Clone this repository on your machine:
git clone https://github.com/hezronkimutai/Questioner-endpoints.git

CD into the project folder on your machine
$ cd Questioner-endpoints

Check out the branch for this PR
$ git checkout ch-directory-structur-163087019

**Relevant pivotal tracker story associated with this PR
#163087019
